src/zampy/recipe.py

```diff
@@ -119,8 +120,11 @@ def run(self) -> None:
             comp = dict(zlib=True, complevel=5)
             encoding = {var: comp for var in ds.data_vars}
             fname = (  # e.g. "era5_2010-2020.nc"
-                f"{dataset_name.lower()}_"
-                f"{self.timebounds.start}-{self.timebounds.end}"
-                ".nc"
+                f"{dataset_name.lower()}_" f"{self.start_year}-{self.end_year}" ".nc"
```

Comment:
why two `f""` and not:
`f"{dataset_name.lower()}_{self.start_year}-{self.end_year}.nc"`